lib/mix/lib/mix/tasks/compile.ex

```diff
@@ -35,6 +35,13 @@ defmodule Mix.Tasks.Compile do
       this has undesirable side-effects (such as skipping some
       compiler checks) and should be avoided.
 
+    * `:prune_code_paths` - prune code paths before compilation. When true
+      (default), this prunes code paths of applications that are not listed
+      in the project file with dependencies.  When false, this keeps the
+      entirety of Erlang/OTP available on the project starts, including
```

Comment:
Should be "on project start" here, or "when the project starts".

Reply:
I literally copied the existing phrase from a few lines below. ;)  Fixing in both places...

Reply:
Eh, too late now, just felt the need to point at the correct English phrasing. Next time!

Reply:
Submitted a separate PR ;)